Exercise_4.java

```diff
@@ -14,6 +41,17 @@ void sort(int arr[], int l, int r)
     { 
 	//Write your code here
         //Call mergeSort from here 
+    		if(l < r) {

```

Comment:
mention the function this code is part of
and try to make your code more readable by explaining your steps